src/Broadcast/InternalState.php

```diff
@@ -44,14 +45,16 @@ final class InternalState
     private StatusInternal $status = StatusInternal::IDLING_BEFORE_GATHERING_PEERS;
     private DeferredCancellation $cancellation;
     private bool $cancelled = false;
+
     public function __construct(
         private int $broadcastId,
         private MTProto $API,
         private Action $action,
         private Filter $filter,
+        private readonly ?float $delay = null,
     ) {
+        Assert::greaterThanEq($this->delay, 0, 'Delay must be greater than or equal to zero');
```

Comment:
if ($this->delay !== null) {
    Assert::greaterThanEq($this->delay, 0, 'Delay must be greater than or equal to zero');
}

Reply:
no need but ok